src/ansys/dpf/core/data_sources.py

```diff
@@ -44,22 +46,28 @@
 from ansys.dpf.core.check_version import version_requires
 from ansys.dpf.core import errors
 
+if TYPE_CHECKING:
```

Comment:
```suggestion
if TYPE_CHECKING:  # pragma: no cover
```
@luisaFelixSalles  To indicate to the coverage tool that this should be ignored. Writing it at the "if" level means every line in the IF is impacted.